index.json

```diff
@@ -239,10 +239,10 @@
   ],
   "fontFamily": {
     "arialBlack": "'Arial Black', 'Arial Bold', Gadget, sans-serif",
-    "lato": "Lato, -apple-system, 'Segoe UI', Helvetica, Arial, sans-serif",
+    "lato": "Lato, -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'",
     "mono": "'Roboto Mono', Monaco, 'Lucida Console', 'Courier New', Courier, monospace",
-    "monoSystem": "Monaco, 'Lucida Console', 'Courier New', Courier, monospace",
-    "sansSystem": "-apple-system, 'Segoe UI', Helvetica, Arial, sans-serif",
+    "monoSystem": "ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace",
```

Comment:
@davidrans, see https://github.com/iFixit/ifixit/pull/37335, https://github.com/iFixit/core-primitives/pull/21, and https://bugs.chromium.org/p/chromium/issues/detail?id=781344&q=BlinkMacSystemFont&can=1&sort=-modified

In short, `BlinkMacSystemFont` causes a whole page reflow hurting LCP, FID and CLS of course only in chrome

Reply:
No good then, that's ~75% of traffic and all those metrics are important to rank, UX, etc

Reply:
I'm trying to trace back what the problem with `BlinkMacSystemFont` was but I'm struggling a bit because the [issue](https://github.com/iFixit/ifixit/pull/37335) is referencing a Dozuki Slack discussion I've no access to.

Moreover, by skimming through the [chromium issue](https://bugs.chromium.org/p/chromium/issues/detail?id=781344&q=BlinkMacSystemFont&can=1&sort=-modified), I have the sensation that they are discussing something slightly different:

- They are profiling an animation of the `<text>` block inside the SVG
- The performance hit seems to be related to the number of fonts specified, not to using `BlinkMacSystemFont` or not using it.

I've seen that there is a [comment](https://bugs.chromium.org/p/chromium/issues/detail?id=781344#c23) from you @jarstelfox but I'm not seeing any CLS by testing the [react-commerce branch](https://react-commerce-git-use-system-font-ifixit.vercel.app/Parts) with the `BlinkMacSystemFont` font family on chrome.

 Also the [other chromium issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1057654#c35) referenced in #21 seems not to be reproducible anymore.

Can you please give me a little bit more of context?

Reply:
>Can you please give me a little bit more of context?

💯 

Part of the problem is that the issue we were seeing on the website doesn't seem to be well documented. I searched multiple repos, slack and emails to work back to the exact error.

From my understanding, we were seeing a large reflow during page loads and all of our digging led us to [the chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=781344&q=BlinkMacSystemFont&can=1&sort=-modified).

We were not sure this was the error:

>Jarred Stelfox
  19:40
Hey I found it. it’s BlinkMacSystemFont
removing that from the critical css fixes the CLS shifting
https://github.com/tailwindlabs/tailwindcss/issues/1402
^ suggested from here

>Ian Rohde
  19:41
:whoa:
That’s part of the primitive stack. "sansSystem": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif", I can make a new version for it later tonight. IIRC we pulled the stack from github, so I’m good with removing it, especially if it’s causing problems

>Jarred Stelfox
  19:48
https://www.google.com/search?q=BlinkMacSystemFont+chrome+bug&oq=BlinkMacSystemFont+[…]g&aqs=chrome..69i57j0i22i30.3051j0j7&sourceid=chrome&ie=UTF-8
A quick google search shows chrome hates that font family
https://bugs.chromium.org/p/chromium/issues/detail?id=1057654#c35
^ found the bug!
Actually, I think it’s: https://bugs.chromium.org/p/chromium/issues/detail?id=781344&q=BlinkMacSystemFont&can=1&sort=-modified
tl;dr: They had a series of rendering bugs that happened a year ago. Now they notice a CLS with it.
One other note: https://bugs.chromium.org/p/chromium/issues/detail?id=554590&q=BlinkMacSystemFont&can=2
It looks like chrome intends to deprecate the font family


I know that thread is not the most helpful, but it seems we were hitting some kind of Layout shifting that went away if we removed the font from the stack. What's tricky is that it's an apple font, so you have to be on a mac only to receive the shifting.

Related, but it looks like the chrome team has put time into fixing bugs with this font overall. Maybe it's safe to use now.

Reply:
`BlinkMacSystemFont` may be safe to use now, but is it reliable long-term? My proposal [here](https://github.com/iFixit/core-primitives/pull/36#issuecomment-1226011159) achieves the same functionality without relying on it.

Reply:
> `BlinkMacSystemFont` may be safe to use now, but is it reliable long-term? My proposal [here](https://github.com/iFixit/core-primitives/pull/36#issuecomment-1226011159) achieves the same functionality without relying on it.

The linked font stack will make MacOS Chrome render either Helvetica or Arial before SF though

> we were hitting some kind of Layout shifting that went away if we removed the font from the stack. What's tricky is that it's an apple font, so you have to be on a mac only to receive the shifting.

If you have a repro we can definitely test it out to check if it is still an issue